rails/app/assets/stylesheets/admin/_main_content_style.scss

```diff
@@ -35,7 +35,7 @@
 .form {
     margin-left: 2rem;
     border: 4px  #0f566b solid;
-    width: 300px;
+    width: auto;
```

Comment:
does this leave a consequence on other admin forms? thats a bit of a sweeping css change